Events

itexpert120 opened the pull request:
Resolves https://github.com/NEARBuilders/gateway/issues/126

elliotBraem commented:
@itexpert120 Looks great, it works very well and I'm glad to see all the filters working. Some feedback:

- [ ] clear filters doesn't clear the date fields (I'm a bit confused by the props.setFilters being passed around from modals -- any way you think to simplify it all? Or reasoning to justify current setup?) 
- [x] I renamed "fetchEvents" to "fetchThings", as it wasn't really specific to events (also renamed to lib.everything-sdk)
- [ ] rename "test" to "every" once ready
- [ ] Imagine if we want others to use this calendar component, how can we have the theme all define in one place for easy customization?

itexpert120 commented:
> @itexpert120 Looks great, it works very well and I'm glad to see all the filters working. Some feedback:
> 
> * [ ]  clear filters doesn't clear the date fields (I'm a bit confused by the props.setFilters being passed around from modals -- any way you think to simplify it all? Or reasoning to justify current setup?)
> * [x]  I renamed "fetchEvents" to "fetchThings", as it wasn't really specific to events (also renamed to lib.everything-sdk)
> * [ ]  rename "test" to "every" once ready
> * [ ]  Imagine if we want others to use this calendar component, how can we have the theme all define in one place for easy customization?

Thanks for the review @elliotBraem. The reason for having setFilter passed to filter Modal is because the filters are stored in parent component. This makes handling filters easier. If you have a better solution, please share and I'll implement it.

elliotBraem commented on apps/builddao/widget/lib/everything-sdk.jsx:
```diff
@@ -0,0 +1,38 @@
+    events.push(JSON.parse(Social.get(key, "final")))
+  );
+
+  return events;
```
rename all "events" to "things"

itexpert120 replied:
kk
